pootle/apps/pootle_store/models.py

```diff
@@ -333,6 +333,37 @@ class Meta:
     ############################ Properties ###################################
 
     @property
+    def edits(self):
+        """Submissions that change this unit's target
+
+        :return: Queryset of `Submissions`s that change this unit's target
+        """
+        return (self.submission_set
+                    .exclude(new_value__isnull=True)
+                    .filter(field__in=SubmissionFields.TRANSLATION_FIELDS)
+                    .filter(type__in=SubmissionTypes.EDITING_TYPES))
+
+    @property
+    def comments(self):
+        """Submissions that change this unit's comment
+
+        :return: Queryset of `Submissions`s that change this unit's comment
+        """
+        return self.submission_set.filter(field=SubmissionFields.COMMENT)
+
+    @property
+    def suggestion_reviews(self):
+        """Submissions from this unit that review (reject/accept) suggestions.
+
+        :return: Queryset of `Submissions`s for this unit that
+            `REJECT`/`ACCEPT` `Suggestion`s
+        """
+        # reject_suggestion does not set field so we must exclude STATE reviews
```

Comment:
I think in my testing there was some submissions that had field set to `SubmissionFields.STATE` but had type in `SubmissionType.REVIEW_TYPES`